### PR TITLE
Add human device gesture interaction(hdgi) ontology re-direction details

### PR DESCRIPTION
**Pull request by madhawap**
- Create hdgi directory
- Add configured .htaccess file
- Add README.md

### CODE REVIEW

File: hdgi/.htaccess [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+RewriteRule ^$ https://madhawap.github.io/human-device-interaction-ontology/v0.1/ontology.nt [R=303,L]
+
+# Rewrite rule to serve TTL content from the vocabulary URI if requested
+RewriteCond %{HTTP_ACCEPT} text/turtle [OR]
+RewriteCond %{HTTP_ACCEPT} text/\* [OR]
+RewriteCond %{HTTP_ACCEPT} \*/turtle 
+RewriteRule ^$ https://madhawap.github.io/human-device-interaction-ontology/v0.1/ontology.ttl [R=303,L]
+
+RewriteCond %{HTTP_ACCEPT} .+
+RewriteRule ^$ 406.html [R=406,L]
```
Comment (dgarijo) [inline hdgi/.htaccess:44]: This is redirecting to a local file that doesn't seem to exist

File: hdgi/.htaccess [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+# Default response
+# ---------------------------
+# Rewrite rule to serve the RDF/XML content from the vocabulary URI by default
+RewriteRule ^$ ontology.xml [R=303,L]
```
Comment (dgarijo) [inline hdgi/.htaccess:49]: This redirects to a file that doesn't seem to exist. Please use the URL above instead (https://madhawap.github.io/human-device-interaction-ontology/v0.1/ontology.xml).

**Comment by dgarijo** [comment]
@madhawap I added a few comments, please address them so your w3id will work correctly. Thanks!
Also, do you mind adding your github id as well in the readme as maintainer? It makes it easier for me to verify.

File: hdgi/.htaccess (hunk 1 above)
Comment (madhawap) [reply hdgi/.htaccess]: Fixed this to https://madhawap.github.io/human-device-interaction-ontology/v0.1/406.html.

File: hdgi/.htaccess (hunk 2 above)
Comment (madhawap) [reply hdgi/.htaccess]: Fixed the default response URL.

**Pull request merged**